src/messages/MMDSLoadTargets.h

```diff
@@ -17,21 +17,21 @@
 
 #include "msg/Message.h"
 #include "mds/mdstypes.h"
-#include "messages/PaxosServiceMessage.h"
+#include "messages/MMDSOp.h"
 #include "include/types.h"
 
 #include <map>
 using std::map;
 
-class MMDSLoadTargets : public PaxosServiceMessage {
+class MMDSLoadTargets : public MMDSOp {
```

Comment:
This is not an MMDSOp (inter-MDS op). These are messages sent from MDS to monitor.

Reply:
```
<jcollin_> hi zyan : What about MMDSLoadTargets? Is it mds-mds messaging? I doubt as it is inheriting from PaxosServiceMessage.
(zyan) mds-mds class
<jcollin_> ok
(zyan) I don't think it needs to be inheriting paxos
<jcollin_> Yeah, that's right. It should be from SafeMessage instead
<jcollin_> Anyway, Ill update it with MMDSOp now.
```
Quoting the above conversation with @ukernel here. Zheng, could you please clarify this?

Reply:
I think Zheng confused it with `MHeartbeat`.